changes/820.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed the handling of missing model
```

Comment:
This text is used as a release note, so it should be written in the past tense, in a way that anyone can understand it without context. Something like:

Android emulators that don't provide a model identifier can now be used to launch apps.